governance/parameter-docs/param-dai-savings-rate.md

```diff
@@ -0,0 +1,53 @@
+# Dai Savings Rate
+
+```
+Alias: DSR
+Parameter Name: Pot_dsr
+Containing Contract: Pot
+Scope: System
+Technical Docs: https://docs.makerdao.com/mcd-developer-guides/developer-guides-and-tutorials#dai-savings-rate-dsr
+```
+
+## Description
+The Dai Savings Rate allows Dai holders to deposit Dai into the contract and earn interest paid in Dai. Maker Governance can alter the rate at which interest accrues.
```

Comment:
Maybe focus this more on the perspective of governance. For example, the first sentence should refer to governance rather than DAI Holders. "The Dai Savings Rate parameter allows Maker Governance to determine the interest rate available to DAI Holders when locking DAI in the DSR smart contract."